documentation/docs/examples/Cognite3DViewerDemo.tsx

```diff
@@ -33,6 +33,7 @@ export default function Cognite3DViewerDemo({ client, modelId, revisionId }: Dem
     }
 
     (window as any).viewer = viewer;
+    (window as any).sdk = client;
```

Comment:
I have written something on it here already, could you please check that review too and maybe apply some changes? https://github.com/cognitedata/reveal/pull/956